controllers/errors.controller.js

```diff
@@ -0,0 +1,9 @@
+exports.handleCustomErrors = (err, req, res, next) => {
+  if (err.status) {
+    res.status(err.status).send(err.msg);
+  }
+};
+
+exports.handle500errors = (err, req, res, next) => {
```

Comment:
I would leave a console.log in here as it will give you more information about the error that is coming in if it hasn't been caught by the previous error handling middleware